### ISSUE TITLE
'Enterprise Policy Generator' update schedule

### ISSUE THREAD

**Issue by StanGets** [cadeyrn/enterprise-policy-generator#55]
Hi,

I read the availability of new [Policy templates for Firefox 63 and 60.3](https://github.com/mozilla/policy-templates/releases) which mentions:

> New policies include:
> 
> DNS Over HTTPS (Firefox 63 only)
> The bookmark policy has been updated to allow 50 bookmarks.
> 
> Be sure to check out about:policies on Firefox 63.

There's also the _'Read certificates from the Windows certificate store' error_ described here at #46 

Do you have any schedule for updating 'Enterprise Policy Generator'? The extension is so handy. Of course I could add manually the latest templates but that would be confusing.

Thanks.

**Comment by cadeyrn** [comment]
Hi,

I'll release an update as soon as I get back home (I'm in London right now).

**Comment by cadeyrn** [comment]
The release is planned for this weekend (or very soon after the weekend if the translations needs more time).

> DNS Over HTTPS (Firefox 63 only)

I included this new policy in the update.

> The bookmark policy has been updated to allow 50 bookmarks.

In the current version there is no restriction for the number of bookmarks in the Enterprise Policy Generator so I don't have to change anything. 

> Be sure to check out about:policies on Firefox 63.

My plan is to add a note about the existence of about:policies in the update after the next update of Enterprise Policy Generator.

> There's also the 'Read certificates from the Windows certificate store' error described here at #46

I added a note that this option only works on Firefox 64 and higher.
